.github/workflows/build_test_deploy.yml

```diff
@@ -164,7 +164,7 @@ jobs:
       - name: Npm install
         run: npm install
       - name: Run cypress
-        run: npm run cy:run -- --env url='${{ secrets.ENDPOINT }}',authorizationHeader='${{ secrets.CYPRESS_TEST_SECRET }}'
+        run: npm run cy:run -- --env grepTags=@dev,grepTags=@stage,url='${{ secrets.ENDPOINT }}',authorizationHeader='${{ secrets.CYPRESS_TEST_SECRET }}'
```

Comment:
In your readme you said you can do - grepTags=@dev,@stage

Reply:
grepTags=@dev+@stage will only run tests with these tags.

grepTags=@dev,grepTags=@stage will run tests both tags with dev & stage. 

Will demo to you guys in a bit

Reply:
you can use grepTags=@dev+@stage, or grepTags=@dev,grepTags=@stage. I've chosen the latter to uncomplicate things - there are actually many combinations, recommend reading the docs to see which usage you guys prefer.